rekolekcje-engine/src/main/resources/application.yml

```diff
@@ -12,7 +12,7 @@ spring:
 
   datasource:
     driverClassName: org.postgresql.Driver
-    url: jdbc:postgresql://localhost:5432/rekolekcjedb
+    url: jdbc:postgresql://localhost:5430/rekolekcjedb
```

Comment:
zmień port na domyślny. Taki setup utrudnia postawienie aplikacji od zera. Albo trzeba instalować dockera i wtedy wszystko się stawia, ale jak ktoś chce po prostu podpiąć sobie lokalnego postgresa, którego kiedyś sobie zainstalował z domyślnymi ustawieniami, to wtedy jest problem, bo musi kombinować, zamiast po prostu założyć bazę i wystartować aplikację. Konfiguracja portów inna niż domyślna powinna być pozostawiona użytkownikowi, który ma jakiś nietypowy setup.

Reply:
ok. Zrób rebase i wrzucimy to :)